### PR TITLE
Added 'Solo Mode,' which lets you only display your own DPS

### LINKED ISSUE #123: Suggestion: Solo mode?
I would like it if there was a setting you could enable to only see your personal DPS, akin to how Kagerou allows that by minimizing it.

### PR DESCRIPTION
**Pull request by Brafficus**
Closes #123

### CODE REVIEW

File: src/Config.js [hunk 1]
```diff
@@ -25,6 +25,8 @@ class ConfigRaw extends Component {
 
     // And then save it to localStorage!
     localStorage.setItem('horizoverlay', JSON.stringify(config))
+
+    console.log(config);
```
Comment (bsides) [inline src/Config.js:29]: Please remove

File: src/CombatantHorizontal.js [hunk 2]
```diff
@@ -36,6 +36,8 @@ export default class CombatantHorizontal extends Component {
     // don't need to render this component if this is a limit break
     if (!data.Job && name === 'limit break') return null
 
+    if (!isSelf && config.enableSoloMode) return null
```
Comment (bsides) [inline src/CombatantHorizontal.js:39]: Please add a comment here like the above

File: src/locale.js [hunk 3]
```diff
@@ -83,6 +84,7 @@ const locale = {
       toggleOption11: 'Max Hit',
       toggleOption12: 'Show "jobless" Combatants',
       toggleOption13: 'Enable streamer mode (blur all names)',
+      toggleOption14: 'Mostre apenas o seu DPS',
```
Comment (bsides) [inline src/locale.js:87]: 'Mostrar apenas o seu DPS'

**Review by bsides — CHANGES_REQUESTED** [review]
If you could please check these, I'd really appreciate. Thank you!

File: src/Config.js (hunk 1 above)
Comment (Brafficus) [reply src/Config.js]: Done!

File: src/CombatantHorizontal.js (hunk 2 above)
Comment (Brafficus) [reply src/CombatantHorizontal.js:40]: Sorry, meant to ask if the comment I added works but accidentally resolved

File: src/locale.js (hunk 3 above)
Comment (Brafficus) [reply src/locale.js]: Switched!

**Review by bsides — APPROVED** [review]

**Comment by bsides** [comment]
@Brafficus thank you for your contribution, please test it throughly and report if any bug appear.

**Pull request merged**